Source/WebCore/editing/cocoa/AttributedString.mm

```diff
@@ -334,6 +347,12 @@
     }
     if ([value isKindOfClass:PlatformNSPresentationIntent])
         return { { { RetainPtr { (NSPresentationIntent *)value } } } };
+#if ENABLE(MULTI_REPRESENTATION_HEIC)
+    if ([value isKindOfClass:PlatformWebMultiRepresentationHEICAttachment]) {
+        auto attachment = static_cast<WebMultiRepresentationHEICAttachment *>(value);
```

Comment:
shouldn’t this use the dynamic objc cast thing we have?

Reply:
Can't for soft-linked classes.